cmd/machineset/main.go

```diff
@@ -152,6 +159,35 @@ func main() {
 		log.Fatal(err)
 	}
 
+	desiredVersion := getReleaseVersion()
+	missingVersion := "0.0.1-snapshot"
```

Comment:
i wonder if it makes sense to turn `0.0.1-snapshot` into a constant value since we also use it `getReleaseVersion` function?

Reply:
Yep, that makes sense to me! :D